tests/test_discovery.py

```diff
@@ -0,0 +1,93 @@
+import re
+
+from tap_tester import connections, menagerie
+from base import DixaBaseTest
+
+
+# import debugpy
+# debugpy.listen(8000)
+# print("Start debugging...")
+# debugpy.wait_for_client()
+
+
+class DixaDiscoveryTest(DixaBaseTest):
+
+    def name(self):
+        return "tap_tester_discovery_test"
+
+    def test_run(self):
```

Comment:
This is missing some of the assertions for a standard discovery test https://github.com/stitchdata/tap-tester/blob/master/reference/expectations/saas_taps.md#assertions